docker-images/openemr/Dockerfile

```diff
@@ -17,14 +17,15 @@ ENV APACHE_LOG_DIR=/var/log/apache2
 COPY php.ini /etc/php7/php.ini
 COPY openemr.conf /etc/apache2/conf.d/
 #add runner and auto_configure and prevent auto_configure from being run w/o being enabled
-COPY run_openemr.sh .
-COPY auto_configure.php .
-RUN chmod 700 run_openemr.sh \
-    && chmod 000 auto_configure.php \
+COPY run_openemr.sh auto_configure.php utilities/unlock_admin.php /var/www/localhost/htdocs/openemr/
```

Comment:
Are you copying these scripts into the webroot at /var/www/localhost/htdocs/openemr/ ?
(if so, that doesn't seem very safe; can they work if placed outside the web path)

Reply:
I thought `sites` below it was the webroot; I must've been mistaken. Isn't this where auto_configure.php goes? I don't know if it'll run when outside the webroot or not -- is there somewhere else you'd like it?

Reply:
Also it seems like the unlock_admin.php is just basically specific to lightsail. Does it make sense to include this in the main docker where it will mostly not be utilized?

Reply:
I considered that, but decided that we're probably going to come up with more random little nubs, and remember that it has to be inside the container to work anyways, so if I don't put it in the Docker now I'll have to inject it from the Lightsail host. It just seemed cleaner to put it here.

Reply:
Also, it's not quite that it's specific to Lightsail -- this isn't code for Lightsail at all, it's for the AWS MArketplace appliance. It's just being built from the Lightsail path.

Reply:
Looked into this more.
auto_configure gets removed after an autoinstall, but is left on a manual install. We need to also remove it for a manual install (otherwise, it is a security risk). and would place  unlock_admin.php in /root/ and run it from there (it will work anywhere).

Reply:
And then if the unlock_admin.php is in /root/ and permissions are locked, then we'd be safe to make it ubiquitous for all dockers.

Reply:
Done.